### PR TITLE
Common jdbi util for backend which used for common functionality of jdbi impl after removing hibernate util

### PR DESCRIPTION
**Pull request by anandjakhaniya**
## Impact and Context

All functions are extracted from CommonHibernateUtils and it is the same one, there is not have any changes

### CODE REVIEW

**Comment by conradoverta** [comment]
>All functions are extracted from CommonHibernateUtils and it is the same one, there is not have any changes

Then let's move instead of copying >300 LOC

**Comment by anandjakhaniya** [comment]
> > All functions are extracted from CommonHibernateUtils and it is the same one, there is not have any changes
> 
> Then let's move instead of copying >300 LOC

that is the new file so all moved code also show as a new lines and show removed code in CommonHibernateUtils. I did move code from `CommonHibernateUtils` to `JdbiUtil` i hope it is same like you are expecting.

**Comment by conradoverta** [comment]
> > > All functions are extracted from CommonHibernateUtils and it is the same one, there is not have any changes
> > 
> > 
> > Then let's move instead of copying >300 LOC
> 
> that is the new file so all moved code also show as a new lines and show removed code in CommonHibernateUtils. I did move code from `CommonHibernateUtils` to `JdbiUtil` i hope it is same like you are expecting.

The PR had no deletions when I commented =)

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonJdbiUtil.java [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.LiquibaseException;
+import liquibase.exception.LockException;
+import liquibase.lockservice.LockServiceFactory;
+import liquibase.resource.FileSystemResourceAccessor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public abstract class CommonJdbiUtil {
```
Comment (conradoverta) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonJdbiUtil.java]:
```suggestion
public abstract class CommonDBUtil {
```
This doesn't seem specific to the database method (jdbi vs hibernate), right?

**Review by conradoverta — APPROVED** [review]

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonJdbiUtil.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonJdbiUtil.java]: Renamed

**Pull request merged**